vendor/plugins/edu.harvard.dce.paella.iFrameApiAppraiser/iFrameApiAppraiser.js

```diff
@@ -103,7 +103,7 @@ paella.addPlugin(function () {
       // Make sure the event is supported
       if (this.eventsToAppraiseWithTime.contains(eventType)) {
         this.sendMessageToEmbedApi(eventType, params.currentTime);
-      } else if (this.eventsToAppraiseWithoutTime) {
+      } else if (this.eventsToAppraiseWithoutTime.contains(eventType)) {
```

Comment:
If this isn't being cross-compiled, in older browsers, it's best to use `indexOf(...) >= 0` instead of `contains`.
But that's probably not an issue because we require modern browsers.

Reply:
I messed up on that.  I fixed by changing the contains to includes.